app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import shutil
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if len(command) == 3 and command[0] == "mv":
+        _, source_file, destination_path = command
+        dir_path = os.path.dirname(destination_path)
+        if dir_path and not os.path.exists(dir_path):
```

Comment:
`If name in destination path ends with / it must be considered as a directory.`
your code should be able to pass `mv file.txt first_dir/second_dir/` and successfully move the file